internal/controller/proxmoxcluster_controller_test.go

```diff
@@ -45,6 +49,104 @@ var (
 	testFinalizer = "cluster-test.cluster.x-k8s.io"
 )
 
+var _ = Describe("Proxmox ClusterReconciler", func() {
+	BeforeEach(func() {})
+	AfterEach(func() {})
+
+	Context("Reconcile an ProxmoxCluster", func() {
```

Comment:
why you can't add those tests cases to the other context?

Reply:
Because I wanted to have a separate, isolated context specific to the test at hand.